stdlib/LinearAlgebra/src/hessenberg.jl

```diff
@@ -1,20 +1,25 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-struct Hessenberg{T,S<:AbstractMatrix{T}} <: Factorization{T}
+struct Hessenberg{T,TH,S<:AbstractMatrix{TH},V<:Number} <: Factorization{T}
     factors::S
-    τ::Vector{T}
+    τ::Vector{TH}
```

Comment:
Might be worth making the vector parametric now that the type is adjusted anyway. Could potentially be useful for DistributedArrays and CuArrays.

Reply:
You mean `τ::Tτ` where `Tτ<:AbstractVector{TH}`?